### ISSUE TITLE
SystemJS 0.17.0+ Support

### ISSUE THREAD

**Issue by scoarescoare** [Workiva/karma-jspm#58]
The new version of systemjs changes things a little bit. 

It is no longer acceptable to change the baseURL after it's already been configured, this is necessary for the normalization process. See here: https://github.com/systemjs/systemjs/issues/514

Because this plugin [defines the baseURL here](https://github.com/Workiva/karma-jspm/blob/45baf1f540c81c3c97db6ed5e40c3af2749b26e5/src/adapter.js#L35) SystemJS is throwing an exception.

**Comment by OliverJAsh** [comment]
See https://github.com/rich-nguyen/karma-jspm

**Comment by OliverJAsh** [comment]
Note we haven't updated the docs, you'll have to look at the code.

**Comment by martinmicunda** [comment]
@OliverJAsh is there plan to merge that into `Workiva/karma-jspm` repo?

**Comment by maxwellpeterson-wf** [comment]
@OliverJAsh we would gladly accept a PR!

**Comment by martinmicunda** [comment]
@OliverJAsh I tried your version [rich-nguyen/karma-jspm](https://github.com/rich-nguyen/karma-jspm) but with SystemJS 0.18.0+ however it doesn't run any tests.

```bash
Chrome 43.0.2357 (Mac OS X 10.10.3): Executed 0 of 0 ERROR (0.003 secs / 0 secs)
```

**Comment by maxwellpeterson-wf** [comment]
Fixed in #61
